### PR TITLE
Win install docs

### PR DESCRIPTION
**Pull request by twangboy**
### What does this PR do?
Removes references to salt-master in the installation and build docs
Adds documentation for building Python 3 installs

### What issues does this PR fix or reference?
https://github.com/saltstack/zh/issues/1187

### Tests written?
NA

### CODE REVIEW

**Comment by rallytime** [comment]
I have switched the base branch on this pull request from `nitrogen` to `2017.7` in preparation for moving from a code-named release to a version number.

File: doc/topics/installation/windows.rst [hunk 1]
```diff
@@ -147,24 +115,17 @@ Here are some examples of using the silent installer:
...
-    Salt-2016.9.1-Setup-amd64.exe /S /master=yoursaltmaster /minion-name=yourminionname /install-master
-
-.. code-block:: bat
-
-    # install only the master but don't start the salt-master service
+    # Install the Salt Minion
+    # Configure the minion and start the service
 
-    *-Setup-*.exe /S /master=yoursaltmaster /minion-name=yourminionname
+    Salt-Minion-2017.5.1-Py2-AMD64-Setup.exe /S /master=yoursaltmaster /minion-name=yourminionname
```
Comment (rallytime) [inline doc/topics/installation/windows.rst]: This version with `2017.5` won't exist. Can you change that to `2017.7`?

File: doc/topics/installation/windows.rst [hunk 2]
```diff
@@ -147,24 +115,17 @@ Here are some examples of using the silent installer:
...
 .. code-block:: bat
 
-    # install the minion and master
-    # configure the minion to talk to the local master
-    # start both services
+    # Install the Salt Minion
+    # Configure the minion but don't start the minion service
 
-    *-Setup-*.exe /S /master=yoursaltmaster /minion-name=yourminionname /start-minion=0
+    Salt-Minion-2017.5.1-Py3-AMD64-Setup.exe /S /master=yoursaltmaster /minion-name=yourminionname /start-minion=0
```
Comment (rallytime) [inline doc/topics/installation/windows.rst]: Same comment here about the version.

File: doc/topics/installation/windows.rst [hunk 3]
```diff
@@ -316,7 +282,7 @@ Go into the ``salt`` directory and checkout the version of salt to work with
 .. code-block:: bat
 
     cd salt
-    git checkout 2016.3
+    git checkout 2017.5.2
```
Comment (rallytime) [inline doc/topics/installation/windows.rst]: Same comment here.

File: doc/topics/installation/windows.rst [hunk 4]
```diff
@@ -367,6 +333,10 @@ with ``pip``
     If ``pip`` is not recognized, you may need to restart your shell to get the
     updated path
 
+.. note::
+    if ``pip`` is still not recognized make sure that the Python Scripts folder
```
Comment (rallytime) [inline doc/topics/installation/windows.rst]: `if` should be `If` :)

File: doc/topics/installation/windows.rst [hunk 5]
```diff
@@ -414,16 +384,20 @@ Navigate to the root ``salt`` directory and install Salt.
 -------------------------------
 
 Navigate to the ``pkg\windows`` directory and run the ``build_pkg.bat``
-with the build version (2016.3) script.
+with the build version (2017.5.2) and the Python version as parameters.
```
Comment (rallytime) [inline doc/topics/installation/windows.rst]: Oh, version here, too. There are a few more references with the `.5` version below. You get the idea. :)

**Review by rallytime — CHANGES_REQUESTED** [review]
I had a couple of small changes that need to be fixed. Then we can get this merged in.

**Review by rallytime — APPROVED** [review]

**Comment by damon-atkins** [comment]
Might be worth adding how to run the windows tests, given this document covers building a developed environment.

**Pull request merged**